workflow/scripts/hic.py

```diff
@@ -64,35 +65,39 @@ def load_hic(
     interpolate_nan=True,
     apply_diagonal_bin_correction=True,
 ):
-    print("Loading HiC")
+    print("Loading HiC Juicebox")
+    HiC_sparse_mat = hic_to_sparse(hic_file, hic_norm_file, hic_resolution)
+    HiC = process_hic(
+        hic_mat=HiC_sparse_mat,
+        hic_norm_file=hic_norm_file,
+        hic_is_vc=hic_is_vc,
+        resolution=hic_resolution,
+        tss_hic_contribution=tss_hic_contribution,
+        window=window,
+        min_window=min_window,
+        gamma=gamma,
+        interpolate_nan=interpolate_nan,
+        apply_diagonal_bin_correction=apply_diagonal_bin_correction,
+        scale=scale,
+    )
+    return HiC
 
-    if hic_type == "juicebox":
-        HiC_sparse_mat = hic_to_sparse(hic_file, hic_norm_file, hic_resolution)
-        HiC = process_hic(
-            hic_mat=HiC_sparse_mat,
-            hic_norm_file=hic_norm_file,
-            hic_is_vc=hic_is_vc,
-            resolution=hic_resolution,
-            tss_hic_contribution=tss_hic_contribution,
-            window=window,
-            min_window=min_window,
-            gamma=gamma,
-            interpolate_nan=interpolate_nan,
-            apply_diagonal_bin_correction=apply_diagonal_bin_correction,
-            scale=scale,
-        )
-        # HiC = juicebox_to_bedpe(HiC, chromosome, args)
-    elif hic_type == "bedpe":
-        HiC = pd.read_csv(
-            hic_file,
-            sep="\t",
-            names=["chr1", "x1", "x2", "chr2", "y1", "y2", "name", "hic_contact"],
-        )
-    elif hic_type == "avg":
-        HiC = pd.read_csv(hic_file, sep="\t", names=["x1", "x2", "hic_contact"])
-        HiC["bin1"] = np.floor(HiC["x1"] / hic_resolution).astype(int)
-        HiC["bin2"] = np.floor(HiC["x2"] / hic_resolution).astype(int)
 
+def load_hic_bedpe(hic_file):
```

Comment:
This and hic_avg are much simpler now. They don't need all the params required for juicebox files